src/test/java/core/mvc/asis/DispatcherServletTest.java

```diff
@@ -27,6 +27,8 @@ void serviceToAnnotationMapping() throws Exception {
         dispatcherServlet.service(request, response);
 
         assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getForwardedUrl()).isNotBlank();
```

Comment:
```suggestion
        assertThat(response.getForwardedUrl()).isEqualTo("home.jsp");
```
위와 같이 정확한 테스트이면 더 좋을 것 같아요.
테스트의 목적이 HomeController에 있는 home() 메서드를 잘 호출하는지 확인하기 위함이니까요.